scripts/quests/otherAreas/Knocking_on_Forbidden_Doors.lua

```diff
@@ -0,0 +1,277 @@
+-----------------------------------
+-- Knocking on Forbidden Doors
+-----------------------------------
+-- Log ID: 4, Quest ID: 78
+-- Enaremand       : !pos
+-- Fyi_Chalmwoh    : !pos -39.273 -16.000 70.126 249
+-- Mire Incense KI : 709
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/mannequins')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local ID = require('scripts/zones/Mhaura/IDs')
+
```

Comment:
Just for consistent styling, `-----------------------------------` and newline after the local ID